index.html

```diff
@@ -205,7 +205,7 @@ <h3 class="cardHeader aboutmeHeader">Skills</h3>
                 <li>Terminal</li>
             </ul>
         </div>
-
+    </div>
     </section>
 
     <section class="contactme centeredGrid">
```

Comment:
- [ ] [OPTIONAL] It seems that your contact me section does not have some margin on the left and right to give it the same width as the width we have for the `my works` section because for now, it is taking the full width, which is not what we have in the Figma design, It can be a great idea to fix this 😉 
![image](https://user-images.githubusercontent.com/55465133/130118557-c2144b77-d586-4653-8d98-230dd36407f4.png)